docs/db/changelog/changesets/user-data-store/1.3.x/20240514-refactor-uds-1.3.0.xml

```diff
@@ -33,8 +33,8 @@
             <column name="encryption_mode" type="VARCHAR(255)" defaultValue="NO_ENCRYPTION">
                 <constraints nullable="false"/>
             </column>
-            <column name="timestamp_created" type="TIMESTAMP WITHOUT TIME ZONE" defaultValueComputed="NOW()"/>
-            <column name="timestamp_last_updated" type="TIMESTAMP WITHOUT TIME ZONE"/>
+            <column name="timestamp_created" type="TIMESTAMP WITHOUT TIME ZONE" defaultValueDate="${now}" />
```

Comment:
@zcgandcomp I am not able to recall the issue with MS SQL, will this default work there?

Reply:
definition in the change log module
 <property name="now" value="current_timestamp" dbms="mssql"/>

and this construct
 <column name="timestamp_created" type="timestamp" defaultValueDate="${now}" />

should work fine

The type="TIMESTAMP WITHOUT TIME ZONE" check our wiki maybe this works, but I'm not sure.

Reply:
It should work, it gets generated as e.g.:

`timestamp_created datetime2 CONSTRAINT DF_uds_photo_timestamp_created DEFAULT GETDATE()`

Reply:
I found that issue https://github.com/liquibase/liquibase/issues/4644